index.html

```diff
@@ -228,8 +228,28 @@ <h3 class="about-list"><span>Skills</span> <i class="fas fa-chevron-right"></i><
         </ul>
     </section>
 
-   
-   </main>   
+  
+  <footer class="contact-section">
+     <div class="contact-container">
+            <h2>Contact me</h2>
+          <div> 
+             <p>If you have an application you are interested in developing,
+              a feature that you need built or a project that needs coding.
+              I’d love to help with it</p> 
+            </div>
+        </div>
+        
+        <form class="input-section">
```

Comment:
- [ ] I have noticed the your form does not have the `action` and `method` attribute which is a project requirement. I am therefore suggesting that your add them to your form so that it can be able to submit data.

- [ ] It would nice if you integrate the form with formspree service. Since this is a static form, this service allows you to receive input that has been entered by the user.

> It can be integrated by setting the action value to `https://formspree.io/f/{form_id}`